service/matching/task_reader.go

```diff
@@ -83,30 +79,8 @@ func newTaskReader(backlogMgr *backlogManagerImpl) *taskReader {
 
 // Start taskReader background goroutines.
 func (tr *taskReader) Start() {
-	if !atomic.CompareAndSwapInt32(
-		&tr.status,
-		common.DaemonStatusInitialized,
-		common.DaemonStatusStarted,
-	) {
-		return
-	}
-
-	tr.gorogrp.Go(tr.dispatchBufferedTasks)
-	tr.gorogrp.Go(tr.getTasksPump)
-}
-
-// Stop taskReader goroutines.
-// Note that this does not wait until
```

Comment:
Not even the author of this comment was willing to wait until they

Reply:
Oh, um, that was me :grimacing:  I must have gotten distracted mid-sentence